shell/models/fleet.cattle.io.gitrepo.js

```diff
@@ -323,14 +324,18 @@ export default class GitRepo extends SteveModel {
   get resourcesStatuses() {
     const clusters = this.targetClusters || [];
     const resources = this.status?.resources || [];
-    const conditions = this.status?.conditions || [];
+    // const conditions = this.status?.conditions || [];
+    // const bundleDeployments = this.bundleDeployments || [];
 
     const out = [];
 
     for (const c of clusters) {
```

Comment:
This code assumes each resource is installed on every cluster. This is not always the case. Users can for example use targetCustomization to deploy resources only to some clusters. Additionally, popular charts install different resources depending on the k8s version of the cluster. Some charts even allow to deploy "extraObjects" via a helm chart value. 🤷